src/js/views/patients/sidebar/action/action-sidebar-activity-views.js

```diff
@@ -157,6 +157,7 @@ const CommentView = View.extend({
       messageHtml: renderTemplate(template, { edited }),
       uiView: this,
       ui: this.ui.edit,
+      shouldShowOnClick: false,
```

Comment:
OK now I'm concerned with this change entirely.
We haven't always had tooltip issues across the app.  So what changed?

But if we remove this, then on a touch device we can't see the edited time?

Reply:
Do we still need to consider touch devices?

Reply:
In which cases? I'm not sure how to determine when the delay should be `0` or `200`.

At this point, my opinion is to revert all these tooltip changes. Get rid of the delay we added in #1519. Which is what's causing these issues. And remove the `shouldShowOnClick` stuff.

The original reason the delay was added is outlined in [this story](https://app.shortcut.com/roundingwell/story/63848/add-a-200ms-delay-to-tooltip-showing-on-hover). It doesn't seem worth the trouble it's causing.